### PR TITLE
Bump to v1.2

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

First time using release.sh script, we'll see what Travis CI thinks.

### CODE REVIEW

File: contrib/rpm/buildah.spec [hunk 1]
```diff
@@ -92,6 +92,85 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
...
+- images: fix usage
+- fix make clean error
+- Change 'registries' to 'container registries' in man
+- add commit test
+- Add(): learn to record hashes of what we add
+- Minor update to buildah config documentation for entrypoint
+- Bump to v1.2-dev
+- Add registries.conf link to a few man pages
+
 * Sat Jun 9 2018 Dan Walsh <dwalsh@redhat.com> 1.2-dev-1
```
Comment (pixdrift) [inline contrib/rpm/buildah.spec]: I think this line needs to be dropped from the changelog.

Comment (TomSweeneyRedHat) [reply contrib/rpm/buildah.spec]: Why does it need to be dropped?  This new release is 1.2-1 and 1.3-dev-1

Comment (pixdrift) [reply contrib/rpm/buildah.spec]: Specifically this line:
`* Sat Jun 9 2018 Dan Walsh <dwalsh@redhat.com> 1.2-dev-1`

It has no changelog entries and the rpm spec delinter probably won't like that.

File: contrib/rpm/buildah.spec [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 Name:           buildah
 # Bump version in buildah.go too
-Version:        1.2
+Version:        1.3
 Release:        2.git%{shortcommit}%{?dist}
```
Comment (pixdrift) [inline contrib/rpm/buildah.spec]: This was also bumped to 2.git by nalind after the shadow-utils change in the Requires. I think it should be bumped back to 1.git with the new version number.

**Comment by rhatdan** [comment]
Replaced by https://github.com/projectatomic/buildah/pull/862

**Pull request closed without merging**